webextensions/api/identity.json

```diff
@@ -45,6 +51,10 @@
                   "version_added": "53",
                   "version_removed": "63",
```

Comment:
This is rendered incorrectly, as "Removed in 63 and later": https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/identity/launchWebAuthFlow#browser_compatibility

Could you change this to `"version_added": 63",` and note that the parameter is optional? It has the same meaning and would be rendered correctly.